thrift/thrift_test.go

```diff
@@ -250,8 +257,9 @@ func TestRegisterPostResponseCB(t *testing.T) {
 		}
 
 		called := make(chan struct{})
-		cb := func(method string, response thrift.TStruct) {
+		cb := func(reqCtx context.Context, method string, response thrift.TStruct) {
 			assert.Equal(t, "Call", method)
+			assert.Equal(t, ctx.Value(key), value)
```

Comment:
Yeah I just wanted to test its the same context being threaded through, don't really mind about where exactly it's created from.